source/_integrations/fyta.markdown

```diff
@@ -0,0 +1,54 @@
+---
+title: Fyta
+description: Instructions on how to integrate Fyta sensors within Home Assistant.
+ha_category:
+  - Sensor
+ha_release: 0.1
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@dontinelli'
+ha_domain: fyta
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The fyta {% term integration %} uses the open API of [FYTA](https://www.fyta.de) to obtain the data from your plant sensors and integrate these into Home Assistant.
```

Comment:
```suggestion
The **FYTA** {% term integration %} uses the open API of [FYTA](https://www.fyta.de) to obtain the data from your plant sensors and integrate these into Home Assistant.
```
What is the integration name exactly? I see fyta, FYTA, and Fyta. Suggest to change it to FYTA, as it is spelled like this on their website. Please correct me if I got this wrong.